pkg/virt-api/webhooks/validating-webhook/admitters/instancetype-admitter_test.go

```diff
@@ -71,6 +82,67 @@ var _ = Describe("Validating Instancetype Admitter", func() {
 		Expect(response.Result.Details.Causes[0].Type).To(Equal(metav1.CauseTypeFieldValueInvalid))
 		Expect(response.Result.Details.Causes[0].Message).To(Equal("dedicatedCPUPlacement is not currently supported"))
 	})
+
+	It("should reject when resources.requests.Memory and spec.Memory are both defined", func() {
```

Comment:
I think you can turn these into a single [`DescribeTable`](https://pkg.go.dev/github.com/onsi/ginkgo/extensions/table#DescribeTable)